extract-doc-modules.sh

```diff
@@ -8,29 +8,88 @@
 # This pulls in all included files.
 # You can exclude unwanted files from the final module list with a sed command.
 
-REPO_PATH=../foreman-documentation/guides
-ASSEMBLY_PATH=$REPO_PATH/common
-FILE=$REPO_PATH/doc-Upgrading_Project_Disconnected/master.adoc
+FILE=
+OUTPUT=
```

Comment:
I would prefer that "OUTPUT" not be a parameter because I want users to be able to run a CQA-check script on the output. Should the file name be hard-coded or can you suggest a better way to integrate the 2 scripts?

At this point, I want to keep them separate so that users also have the option to run Vale on the output file.

Reply:
Makes sense. Let's do that.